project/Component/nef/API.swift

```diff
@@ -0,0 +1,102 @@
+//  Copyright © 2019 The nef Authors.
+
+import Foundation
+@_exported import NefModels
+
+import Bow
+import BowEffects
+
+
+public enum Render: RenderAPI, RenderFP { case build
+    public enum Page: PageAPI, PageFP {   case build
+    }
+}
+
+
+public protocol RenderAPI {
```

Comment:
I don't think we should be providing a non-functional API, and even less doing this distinction in naming. I'd just provide an API that is based on Bow Effects. Moreover, the non-functional API can always be derived from the functional one, so whoever wants it, can easily build it themselves.